contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -82,16 +98,23 @@ contract RiskFundConverter is AbstractTokenConverter {
             "ReserveHelpers: The pool doesn't support the asset"
```

Comment:
This `require` is going to fail when the Comptroller is from the Core pool, because the Pool registry doesn't track the core markets.

Moreover, I would check (here or in the new `getPools` function), if there is really a market in the Core pool for the provided asset. Otherwise, assuming the Comptroller of the core pool, is valid for example for BNBx, we can have negative side effects

Reply:
e1bd56299137a6131b5cf52670df1248c5aec6c1